app/src/main/res/layout/fragment_transactions.xml

```diff
@@ -23,4 +23,4 @@
 
     </LinearLayout>
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+</androidx.coordinatorlayout.widget.CoordinatorLayout>
```

Comment:
What issue was this triggering? Seems like you have 1 level of unnecessary nesting here.

Reply:
As the `LinearLayout` has the same height and width as the parent.

Reply:
confirmation snackbar informing user about successful owner import has to be shown above the bottom bar